types/genesis.go

```diff
@@ -37,12 +37,13 @@ type GenesisValidator struct {
 
 // GenesisDoc defines the initial conditions for a tendermint blockchain, in particular its validator set.
 type GenesisDoc struct {
-	GenesisTime     time.Time                `json:"genesis_time"`
-	ChainID         string                   `json:"chain_id"`
-	ConsensusParams *tmproto.ConsensusParams `json:"consensus_params,omitempty"`
-	Validators      []GenesisValidator       `json:"validators,omitempty"`
-	AppHash         tmbytes.HexBytes         `json:"app_hash"`
-	AppState        json.RawMessage          `json:"app_state,omitempty"`
+	GenesisTime      time.Time                `json:"genesis_time"`
+	ChainID          string                   `json:"chain_id"`
+	GenesisChainLock tmproto.ChainLock        `json:"genesis_chain_lock"`
```

Comment:
is this ChainLock what we got from `InitChain` response?

Reply:
We define genesis params currently in JSON config file. How we should put chain lock there? https://github.com/dashevo/mn-bootstrap/blob/4275cdf8d9bc5253413c6fc087308d648e0fcffc/configs/evonet/drive/tendermint/genesis.json